spec/lib/miq_automation_engine/models/miq_ae_yaml_import_gitfs_spec.rb

```diff
@@ -4,12 +4,20 @@
     @ae_db_dir = Dir.mktmpdir
     @default_hash = {:a => "one", :b => "two", :c => "three"}
     @dirnames = %w(A B c)
-    @domain = "ManageIQ"
-    @domain_file = "#{@domain}/#{MiqAeYamlImportExportMixin::DOMAIN_YAML_FILENAME}"
     @namespaces = %w(NS1 NS2)
     @classes    = %w(CLASS1 CLASS2)
     @instances  = %w(INSTANCE1 INSTANCE2)
     @methods    = %w(METHOD1 METHOD2 METHOD3)
+  end
+
+  def build_git_repository(domain_name, domain_dir)
+    @domain = domain_name
+    @domain_dir = domain_dir
```

Comment:
Oh this is spec specific...I couldn't tell upon first review...

This is fine.